site/assets/js/FixedHeader-0.1.1.js

```diff
@@ -1,34 +0,0 @@
-"use strict";
-
-//noinspection JSUnusedAssignment
-var FusionAuth = FusionAuth || {};
-
-FusionAuth.FixedHeader = function() {
-  Prime.Utils.bindAll(this);
-  this.header = Prime.Document.queryFirst('header.page-header');
```

Comment:
This class still exists in `_header.liquid`, do we know for sure this isn't being used? It looks like this selector would fine the header correctly and bind a scroll events.

Reply:
It looks like it is still used to add/remove a gray bar as you scroll. I added it back.